docs/src/appendix/cast/account/add.md

```diff
@@ -30,9 +30,16 @@ Specify account deployment status as deployed.
 If not passed, sncast will check whether the account is deployed or not.
 
 ## `--private-key <PRIVATE_KEY>`
-Required.
+Optional.
 
 Account private key.
+Required if `--private-key-file` is not passed.
+
+## `--private-key-file <PRIVATE_KEY_FILE_PATH>`
+Optional.
+
+Path to the file holding account private key.
+Required if `--private-key-file` is not passed.
```

Comment:
I think it's more appropriate to keep it near the information mentioning if the argument is optional or not

```suggestion
Optional. Required if `--private-key-file` is not passed.

Account private key.

## `--private-key-file <PRIVATE_KEY_FILE_PATH>`
Optional. Required if `--private-key-file` is not passed.

Path to the file holding account private key.
```